### ISSUE TITLE
[bug] Django-internationalflavor fails to import with new pip resolver

### ISSUE THREAD

**Issue by Griatch** [ralphje/django-internationalflavor#16]
In October 2020, `pip` will switch to a new resolver and one can test it already now with `--use-feature=2020-resolver`. This fails horribly on `django-internationalflavor` and the cause appears to be a packaging error (wrong metadata in package): This works fine with the old `pip` resolver but the new one is stricter. 

```
pip install django-internationalflavor --use-feature=2020-resolver
0-resolver
Collecting django-internationalflavor
  Downloading django-internationalflavor-0.4.0-1.tar.gz (12.4 MB)
     |████████████████████████████████| 12.4 MB 15.8 MB/s 
ERROR: Requested django-internationalflavor from 
https://files.pythonhosted.org/packages/c7<hash>/django-internationalflavor-0.4.0-1.tar.gz#sha256=<hash> 
has different version in metadata: '0.4.0'
```

**Comment by ralphje** [comment]
When doing a new release this should be fixed, the 0.4.0-1 release was only issued due to a packaging issue.

**Comment by Griatch** [comment]
@ralphje Looking forward to the new release then, so we can verify our builds using the new `pip` along with this package. :)
